File: modulo1/segmento3/apunte2/Ejercicio2.java
import java.util.Scanner;

public class Ejercicio2 {
	public static void main(String args[]){
		Scanner keyboard = new Scanner(System.in);

		System.out.print("Ingrese su nombre: ");
		String name = keyboard.nextLine();

		System.out.print("Ingrese la contraseña : ");
		Integer password = Integer.parseInt(keyboard.nextLine());

		if(name.equals("juan") && password == 1234 || name.equals("pedro") && password == 5678) {
			System.out.println("Hola " + name);
		}else{
			System.out.println("Usuario invalido!");
		}
	}
}

File: modulo1/segmento3/apunte2/ejercicio2.web/ejercicio2.js
function ejercicio2(){
	const name = prompt('Ingrese su nombre');
	const password = prompt('Ingrese password: ');

	if(name == 'juan' && password == '1234' || name == 'pedro' && password == '5678'){
		alert(`Hola ${name}`);
	} else {
		alert('Error! USUARIO INVALIDO');
	}
}

